ldes-server-integration-test/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/resultactionsextensions/ResponseToModelConverter.java

```diff
@@ -18,7 +19,7 @@ public ResponseToModelConverter(MockHttpServletResponse response) {
     public Model convert() throws UnsupportedEncodingException {
         return RDFParser
                 .fromString(response.getContentAsString())
-                .lang(RDFLanguages.contentTypeToLang(response.getContentType()))
+                .lang(RDFLanguages.contentTypeToLang(ContentType.create(response.getContentType())))
```

Comment:
This looks hacky.. The method hasn't changed and yet you're now making a new instance of this contentType ?